server/src/main/java/org/elasticsearch/rest/RestHandler.java

```diff
@@ -76,8 +73,8 @@ default boolean allowSystemIndexAccessByDefault() {
         return false;
     }
 
-    default MediaTypeRegistry<? extends MediaType> validAcceptMediaTypes() {
-        return XContentType.MEDIA_TYPE_REGISTRY;
```

Comment:
Just for my own learning, was nobody using this interface method before? Do we need to still report somewhere that the default supported media type is MEDIA_TYPE_REGISTRY?